packages/react-components/src/molecules/__tests__/TagList.test.tsx

```diff
@@ -29,6 +30,12 @@ it('hides tags when there are none', () => {
   expect(queryAllByRole('list')).toEqual([]);
 });
 
+it('disables tag when disabled', () => {
```

Comment:
no shit :D
```suggestion
it('disables its tags when disabled', () => {
```
idk this may make the title a bit less awkward ^^